test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -28,6 +28,7 @@
 
 #include "call_log_manager.h"
 
+#include <cassert>
```

Comment:
We should include `assert.h` in c++ if I am not mistaken. Please double-check this.

Reply:
Any reason why to use assert.h over cassert

Reply:
Sorry, it is the opposite, we can stick to <cassert>